pkg/varcontext/varcontext.go

```diff
@@ -58,6 +58,7 @@ func (vc *VarContext) GetInt(key string) (res int) {
 		return err
 	})
 
+
```

Comment:
Double checking that this added whitespace was intentional.

Reply:
It was not, just a merge mistake.